README.md

```diff
@@ -35,18 +49,73 @@ poetry install
 To use GPU, the correct `torch` and `torch_geometric` version should be installed depending on your host device. You may first need to `pip uninstall` these packages, provided by requirements.
 Then follow installation instructions provided on [torch](https://pytorch.org/get-started/locally/) and [torch_geometric](https://pytorch-geometric.readthedocs.io/en/stable/install/installation.html#installation-from-wheels) websites.
 
+## Repo structure
+
+- **AIAgent** contains Python agent and related infrastructure to train network, prepare data, etc.
+- **GameServers** contains (as submodules) different symbolic execution engines extended to communicate with the agent, generate data for training, etc.
+- **maps** contains target projects that used as inputs for symbolic execution engines, as data for training.
+
 ## Usage
 
+### Follow these steps:
```

Comment:
This section is confusing, as it is unclear, what each step means. In my opinion, it should look smth like this:

1) Build all dependencies.
    - Optionally add new maps to [maps](./maps/) and do 
        ```sh
         make build_SVMs build_maps STEPS_TO_SERIALIZE=<MAX_STEPS>
        ```
2) Using [`example from workflow`](`./workflow/dataset_for_tests_java.json`) as a template, create your own configuration with maps to use for training.
3) Run training process (checkout [this chapter](#run-training)).
4) Use [`onyx.py`](#onnx-conversion) command line tool convert your PyTorch model to ONNX format.
5) Use your ONNX model to guide symbolic execution. (checkout `Running with a trained model`, [integration chapter](#integrate-a-new-symbolic-machine)) or use existing extension to one of SVMs in this repo:
    - Place your model in `./VSharp/VSharp.Explorer/models/model.onnx`
    - run `dotnet GameServers/VSharp/VSharp.Runner/bin/Release/net7.0/VSharp.Runner.dll --method BinarySearch maps/DotNet/Maps/Root/bin/Release/net7.0/ManuallyCollected.dll --timeout 120 --strat AI --check-coverage`

Reply:
Yes, I think this version is more clear. But what is a `Running with a trained model`? 
Also I think we need to extend it with description of running with optuna.

Reply:
> what is a `Running with a trained model`?
maybe it was a name of a header, can't remember. Fixed suggestion

> Also I think we need to extend it with description of running with optuna.
optuna is a part of training configuration, so in my opinion, optuna description should be a part of a section, that describes training configuration